.env.development

```diff
@@ -0,0 +1,4 @@
+APP_HOST=0.0.0.0
```

Comment:
:axe: This is meant for your personal development only (see above) so doesn't need to be checked in.

:bulb: The best way to avoid this is to add `*.env*` to your [Git Global Ignore](https://alchemists.io/articles/git_ignore#_global) file. This is probably the *most important* to add to your global configuration because these usually contain passwords and secrete keys in some applications.